content/good-practices.md

```diff
@@ -1,3 +1,188 @@
 ---
+source: https://github.com/cucumber/cucumber/wiki/Step-Organization/
 title: Good Practices
```

Comment:
Perhaps out of scope but I think it should be best practices. There can be multiple good practices but only one is best.